lib/uwb/UwbSession.cxx

```diff
@@ -11,7 +11,8 @@
 using namespace uwb;
 using namespace uwb::protocol::fira;
 
-UwbSession::UwbSession(std::weak_ptr<UwbSessionEventCallbacks> callbacks, uwb::protocol::fira::DeviceType deviceType) :
+UwbSession::UwbSession(uint32_t sessionId, std::weak_ptr<UwbSessionEventCallbacks> callbacks, uwb::protocol::fira::DeviceType deviceType) :
+    m_sessionId(sessionId),
     m_uwbMacAddressSelf(UwbMacAddress::Random<UwbMacAddressType::Extended>()),
     m_callbacks(std::move(callbacks)),
     m_deviceType{ deviceType }
```

Comment:
Can you re-order these so their order here matches their declaration order? Regardless of the order they appear here, they will always be initialized in the order they're declared, so if they appear out-of-order here, someone might think they're initialized in that order.

I think just `m_deviceType` needs to go first and all else look in order.

```suggestion
    m_deviceType{ deviceType },
    m_sessionId(sessionId),
    m_uwbMacAddressSelf(UwbMacAddress::Random<UwbMacAddressType::Extended>()),
    m_callbacks(std::move(callbacks))

```